src/utils/helpers/translations/mapColumnsByRole.ts

```diff
@@ -0,0 +1,106 @@
+export enum APPLICATION_COLUMN {
+  LAST_ACTIVE_DATE = 'Last active date',
+  DEADLINE_DATE = 'Deadline date',
+  APPLICATION_NAME = 'Application name',
+  APPLICANT = 'Applicant',
+  COMPANY = 'Company',
+  CONSOLIDATOR = 'Consolidator',
+  STAGE = 'Stage',
+  STATUS = 'Status',
+  ACTIONABLE = 'Actionable',
+}
+
+interface ColumnDetails {
+  headerName: string
+  filters: null | string | Array<string>
+  render?: Function
+}
+
+const allColumns: Array<ColumnDetails> = [
+  {
+    headerName: APPLICATION_COLUMN.LAST_ACTIVE_DATE,
+    filters: 'last-active-date',
+  },
+  {
+    headerName: APPLICATION_COLUMN.DEADLINE_DATE,
+    filters: 'deadline-date',
+  },
+  {
+    headerName: APPLICATION_COLUMN.APPLICATION_NAME,
+    filters: 'search',
+  },
+  {
+    headerName: APPLICATION_COLUMN.APPLICANT,
+    filters: 'applicant',
+  },
+  {
+    headerName: APPLICATION_COLUMN.COMPANY,
+    filters: 'org',
+  },
+  {
+    headerName: APPLICATION_COLUMN.CONSOLIDATOR,
+    filters: 'consolidator',
+  },
+  {
+    headerName: APPLICATION_COLUMN.STAGE,
+    filters: ['search', 'stage'],
```

Comment:
Since this is an array of filters, wouldn't it be simpler to make them all arrays, even if only one item? Saves having to do an additional type check when interpreting them later on, no?

Reply:
Sure. I'll review this while I'm thinking about the next issue #207